### DISCUSSION TITLE
Can't seem to get tox to print pip installation progress

### DISCUSSION THREAD

**Discussion by nilansaha** [tox-dev/tox#2108] [Q&A]
I want tox to show progress while its installing packages from the requirements in a setup.py file which it does not show. Any help is appreciated.

Example

`tox.ini`
```
[tox]
envlist = py38

deps = 
    pytest

commands =
   pytest
```
`setup.py`
```
from setuptools import setup, find_packages

with open("requirements.txt") as f:
    requirements = f.readlines()

setup(
    name="sample_project",
    version="0.0.1.dev0",
    include_package_data=True,
    packages=find_packages(),
    install_requires=requirements,
)

```

`requirements.txt`
```
requests==2.26.0
uvicorn==0.14.0
```

In this case tox won't show the pip output while its installing `requests` and `uvicorn` so its difficult to keep track of progress when there is a lot of dependencies in the setup

**Comment by jugmac00** [comment]
`tox -vv` should do the trick.

Reading `requirements.txt` in a `setup.py` is rather unusual. 

The former is used to install packages, where you usually pin all dependencies, even to a minor version number.

The latter is used to install libraries, where you only list the direct dependencies, and where you do not pin them, but only exclude known incompatible versions.

Sometimes, still both are used, but then again, in setup.py are only the direct dependencies with no exact pin, and you use eg pip-tools to generate the requirements.txt.
